frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/vm-wrapper.ts

```diff
@@ -43,7 +43,7 @@ export class VMWrapper extends K8sResourceWrapper<VMKind, VMWrapper> implements
   getOperatingSystem = () => findKeySuffixValue(this.getLabels(), TEMPLATE_OS_LABEL);
   getWorkloadProfile = () => findKeySuffixValue(this.getLabels(), TEMPLATE_WORKLOAD_LABEL);
   getFlavor = () => findKeySuffixValue(this.getLabels(), TEMPLATE_FLAVOR_LABEL);
-
+  getEvictionStrategy = () => this.data.spec?.template?.spec?.evictionStrategy;
```

Comment:
IMHO it will be nice to use types (and fix the VMISpec spec with optional fields):
```
getVirtualMachineInstanceSpec = (): VMISpec => this.data.spec?.template?.spec;
getEvictionStrategy = (): string => getVirtualMachineInstanceSpec()?..evictionStrategy;
```